_posts/2019-07-12-barcelona-announcement.md

```diff
@@ -0,0 +1,79 @@
+
+## Save the date!
+
+We have finally a date for this year's edition!
+
+Rustfest 2019 will start on **November 9th** (Saturday) and will end on **November 12th** (Tuesday).
+
+As usual, this edition will have two first days full of talks and workshops and two additional days (*aka Impl Days*) where you will be able to get together with other rustaceans, meet in smaller groups and work on your own community projects.
+
+## Mixing things a little bit
+
+Although we are keeping the same length in terms of days we have decided to change things a little bit and try something different.
+
+This year, the conference format will change and instead of a first day of talks and a second day of workshops we're going to try to split the days into talks in the morning and workshops in the afternoon.
+
+We took into account the feedback we received from previous editions and we believe this can work well for most of you. Indeed, this will allow us to have more talks and workshops and reduce the intensity of a monothematic format day.
+
+## Venue
+
+Rustfest Barcelona 2019 is going to be hold in the [Vèrtex building at Campus Nord](https://www.upc.edu/campusnord/ca/espais/auditori) belonging to [UPC - Universitat Politècnica de Catalunya](https://www.upc.edu/en).
+
+![Vèrtex Auditorium](/assets/posts/barcelona-vx_auditori.jpg)
+
+We'll be using the two level **Vèrtex auditorium** with capacity for 477 people and access for people with reduced mobility at any of the two levels which we think will make a great venue for the amazing talks we'll be hosting this year. ;D
+
+The workshops and the *Impl Days** will take place in some of the [university's classrooms](https://www.upc.edu/campusnord/ca/espais/aules-vertex) which are fully equiped to host workshops of almost any kind.
+
+We will have also available a chilling area for resting between sessions.
+
+The venue is wheelchair accessible over the lateral street and can be reached easily by public transport. The best options, if you come from Sants Estació, are the [Av. Pedralbes - Cavallers bus station](https://goo.gl/maps/QeNjywZ749Njur8S8) and the [Av d’Esplugues - Av Pearson bus station](https://www.google.com/url?q=https://www.google.com/maps/dir/Sants%2BEstaci%25C3%25B3,%2BBarcelona/Av%2Bd%27Esplugues%2B-%2BAv%2BPearson,%2B08034%2BBarcelona/@41.386477,2.1192745,15z/data%3D!4m15!4m14!1m5!1m1!1s0x12a4987e1c029355:0xb19606f1fb16429e!2m2!1d2.1409712!2d41.3816151!1m5!1m1!1s0x12a4984e1e97de21:0x992b87b41d670e10!2m2!1d2.111625!2d41.3921394!3e3!5i3!5m1!1e2&sa=D&ust=1562873148055000&usg=AFQjCNE1co3JRi_anes-tAbTSK8FFGFX4g); also there’s a subway option to [Palau Reial station](https://www.google.com/url?q=https://goo.gl/maps/4u1dJbpRUHRrrmxE8&sa=D&ust=1562873148056000&usg=AFQjCNHwY0Vd1hK9hNsxd68fmNxsLdHs1A) if you want a quicker route that will require some walk.
```

Comment:
This might be a useful resource that people want to look up, maybe you can link it in the blog post:
https://www.barcelona-tourist-guide.com/en/transport/disabled/access-barcelona-disabled.html

Reply:
done!